pkg/backup/item_backupper_test.go

```diff
@@ -131,6 +131,21 @@ func Test_zoneFromPVNodeAffinity(t *testing.T) {
 			wantKey:   "topology.disk.csi.azure.com/zone",
 			wantValue: "us-central",
 		},
+		{
+			name: "Volume with multiple valid keys, and provider is gke, returns the all match", // it should never happen
```

Comment:
The comment is misleading, instead you should provide the concrete use case when testing the regional disk?